Compiler/script2/cc_symboltable.cpp

```diff
@@ -564,7 +571,7 @@ AGS::Vartype AGS::SymbolTable::VartypeWithArray(std::vector<size_t> const &dims,
         aoa_dims.reserve(old_dims.size() + dims.size());
         aoa_dims = dims;
         aoa_dims.insert(aoa_dims.end(), old_dims.begin(), old_dims.end());
-        // Cut off the first '[�]', it will be replaced by the index of the joint array
+        // Cut off the first '[�]', it will be replaced by the index of the joint array
```

Comment:
Is it intended to insert these symbols here? Not sure if it's UTF-8 or another encoding, but perhaps it's better to replace with 3 dots.

Reply:
I don't know what the alien head means there in the other file. I am using CLion on Linux and it keeps complaining, ONLY in the new compiler files, that the files isn't UTF-8 and that it doesn't know the right encoding.

Reply:
oh, now I noticed you saw three dots, I see them on macOS but on Linux it was showing an Alien Head emoji

Reply:
fixed by using the three dots suggested - now I understand the suggestion.